python/rapidsmpf/rapidsmpf/statistics.pxd

```diff
@@ -25,6 +25,9 @@ cdef extern from "<rapidsmpf/statistics.hpp>" nogil:
         bool is_memory_profiling_enabled() except +
         unordered_map[string, cpp_MemoryRecord] get_memory_records() except +
 
+        @staticmethod
+        shared_ptr[cpp_Statistics] disabled() except +
```

Comment:
do we really need to expose this to python? Its the same as `Statistic(False)` isnt it?

Reply:
Sorry, I thought I intended to delete this. I agree it's not necessary.